move "lang_list" to options.cpp, remove unused code

laroche opened the pull request:
"lang_list" is only used in options.cpp, so move it there. At the moment it is also defined "static"
in a header file, so this var is instantiated in each source file that includes "options.h".

"l_bOsLike" in download_mgr.cpp is an unused var.

"onTestSoundFinished()" in options.cpp is an unsed function.

best regards,

Florian La Roche

leamas commented:
Hey,

I'm not the maintainer, and doesn't speak for anyone but myself. That said, if you rebase this to current master it looks like a valuable and clean patch to me.

leamas commented:
BTW: I think you just could drop the onTestSoundFinished() rather than using #ifdef 0. It's just a relic.

laroche commented:
Rebased to newest upstream master and removed onTestSoundFinished() completely
(plus also the var ptest_sound which is also completely unused).

best regards,

Florian La Roche
